x-pack/solutions/observability/plugins/observability/public/pages/alerts/alerts.test.tsx

```diff
@@ -110,6 +117,18 @@ jest.mock('../../hooks/use_has_data', () => ({
 const { useTimeBuckets } = jest.requireMock('../../hooks/use_time_buckets');
 const { useHasData } = jest.requireMock('../../hooks/use_has_data');
 
+jest.mock('../../hooks/use_get_available_rules_with_descriptions');
```

Comment:
To avoid following error:

![image](https://github.com/user-attachments/assets/5d45e3ea-2b20-4260-821c-7f28ab3fd050)